### DISCUSSION TITLE
how to profile wasm in wasmedge?

### DISCUSSION THREAD

**Discussion by giteewif** [WasmEdge/WasmEdge#3856] [Q&A]
is wasmedge support perf, and could see the func execution?
![image](https://github.com/user-attachments/assets/6ce78cf2-32e2-40a7-b6f6-ea93b7685ac6)

**Comment by hydai** [comment]
Currently, wasmedge doesn't support this kind of information. Please feel free to create an issue to propose a profiling way you want.
